File: multi-language/nodejs/circle.ts

class Circle {  
    Radius: number;  
    constructor(Radius: number) {  
        this.Radius = Radius;  
    }  
    Area() {  
        return Math.PI * this.Radius * this.Radius;  
    }  
    Perimeter() {  
        return 2 * Math.PI * this.Radius;  
    }  
    Summary() {

        var formatted = `area=${this.Area()}, perimeter=${this.Perimeter()}`;
        return formatted;
    }
}  

const args = require('yargs').argv;

var radius = args.radius
var circle = new Circle(radius);  

console.log(circle.Summary())

File: multi-language/python/circle.py
import math

class Circle:
    def __init__(self, radius):
        self.radius = radius

    def area(self):
        return math.pi * self.radius * self.radius
    
    def perimeter(self):
        return 2.0 * math.pi * self.radius

    def summary(self):
        return ("area={area}, perimeter={perimeter}".format(area=round(self.area(),2), perimeter=round(self.perimeter(),2)))

